lib/utils.js

```diff
@@ -199,6 +199,9 @@ exports.encode = function encode (value) {
   else if (typeof value === 'string') {
     return '"' + exports.escape(value) + '"';
   }
+  else if (value instanceof Date) {
+    return value.getTime();
```

Comment:
This will only work in schemaful mode, in schemaless mode it'll insert a numeric value. I think we need to return a string in this format: `"2015-01-01 10:00:00"`